src/utilities/music/MusicManager.java

```diff
@@ -73,56 +77,60 @@ public synchronized void emptyPlayer(BotCommand command){
 	public void loadTrack(final BotCommand command, final String source){
 		
 		MusicPlayer player = getPlayer(command);
-		
+
+
 		command.getGuild().getAudioManager()
 				.setSendingHandler(player.getAudioHandler());
 		
+		
 		manager.loadItemOrdered(player, source, new AudioLoadResultHandler(){
-			
```

Comment:
I feel like the changes made to `MusicManager` are only either through tests and formatting. I'm confident enough that the format was already well established before this PR and so I believe you should [revert the changes on MusicManager](https://stackoverflow.com/a/43275994).

If there's something I'm missing, just say so and I'll remove this change request.